aws-dual-stack/openshift-upgrade-cluster.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -Eueo pipefail
+
+usage() {
+  cat <<EOF_USAGE
+Created by Aspen Mesh:
+This script adds IPv6 overlay networking to an existing ipv4 OpenShift Cluster in AWS. Waits for all OpenShift Network pods
+to be upgraded before completion.
+Requires: kubectl, openshift (cluster, installer, client, pull secret), aws cli and creds
+Tested on OSX 11.6
+
+Usage: ${scriptName} <cluster_name>
+Arguments:
+                cluster_name: [string]
+EOF_USAGE
+  exit 1
+}
+
+if [ $# -lt 1 ]; then
+  usage
+  exit 1
+fi
+
+
+cluster=$1
+workdir=_install_
+
+## Global vars
+INSTALL_DIR="${workdir}"
+export KUBECONFIG="$INSTALL_DIR/auth/kubeconfig"
+export OC_CMD="$INSTALL_DIR/oc"
+export KC_CMD="$INSTALL_DIR/kubectl"
```

Comment:
Are these installed to the install_dir or is this an assumption you made? I thought they'd be in the PATH of the user

Reply:
both of these come from the openshift installer and client downloads.